source/billing.txt

```diff
@@ -46,33 +47,33 @@ how much you pay each month:
 
 .. note:: Track Your Usage
    
-   You can track an application's usage for the current month from the
+   You can track an application's usage for the current day from the
    application dashboard in the App Services UI.
 
-Monthly Free Tier
+Daily Free Tier
 ~~~~~~~~~~~~~~~~~
 
-All App Services Apps in a MongoDB Atlas project share a single monthly free
-tier. All usage below the free tier thresholds in a given month is not billed.
-As soon as a project exceeds any monthly free tier threshold, App Services starts 
+All App Services Apps in a MongoDB Atlas project share a single daily free
+tier. All usage below the free tier thresholds in a given day is not billed.
+As soon as a project exceeds any daily free tier threshold, App Services starts 
 billing for additional usage of any kind for that project.
 
 Free tier thresholds
 ````````````````````
 
-App Services is free to use below the following monthly free tier thresholds:
+App Services is free to use below the following daily free tier thresholds:
 
-- 1,000,000 :ref:`requests <billing-requests>` or 500 hours of
-  :ref:`compute <billing-compute>` or 10,000 hours of :ref:`sync
+- 50,000 :ref:`requests <billing-requests>` or 25 hours of
+  :ref:`compute <billing-compute>` or 30,000 minutes of :ref:`sync
   <billing-sync>` runtime (whichever occurs first)
 
-- 10GB of :ref:`data transfer <billing-data-transfer>`
+- 0.5 GB of :ref:`data transfer <billing-data-transfer>`
```

Comment:
Technical Reviewer, is data transfer measured in GB or GiB? Said GiB on the PD, but I see GB in the App Services UI.

Reply:
Customer facing its GB, internally we use GiB and GB interchangeably across Atlas